src/main/java/org/mule/service/http/impl/service/server/grizzly/GrizzlyHttpServer.java

```diff
@@ -60,6 +60,7 @@
 
   private volatile int openConnectionsCounter = 0;
   private final Object openConnectionsSync = new Object();
```

Comment:
move this inside `CountAcceptedConnectionsProbe` and refactor to use an AtomicInteger